Classes/MasterLooterUI.lua

```diff
@@ -307,7 +307,8 @@ function MasterLooterUI:draw(itemLink)
                     end
 
                     local osRoll = selected.cols[4].value == "OS";
-                    return GL.RollOff:award(selected.cols[1].value, GL.Interface:getItem(self, "EditBox.Item"):GetText(), osRoll);
+                    local stackedRoll = selected.cols[4].value == GL.Settings:get("StackedRoll.identifier", "SR");
```

Comment:
Reminder for me to check this again after we've settled on a name for this feature, since `SR` is the commonly accepted abbreviation for soft-reserve.